mypy_django_plugin/transformers/fields.py

```diff
@@ -150,6 +152,22 @@ def set_descriptor_types_for_field(
         is_set_nullable=is_set_nullable or is_nullable,
         is_get_nullable=is_get_nullable or is_nullable,
     )
+
+    # reconcile set and get types with the base field class
+    base_field_type = next(base for base in default_return_type.type.mro if base.fullname == FIELD_FULLNAME)
```

Comment:
```suggestion
    base_field_type = next(base for base in default_return_type.type.mro if base.fullname == fullnames.FIELD_FULLNAME)
```

Since we already have `fullnames` imported